README.md

```diff
@@ -9,20 +9,45 @@ This app is a course project of <https://ucsb-cs156.github.io>, a course at [UC
+services that require configuration
+* Auth0.com (for authentication)
+* Google (for authentication)
+* A postgres database provisioned on Heroku
+
+All of the setup steps for running the app on localhost and Heroku are described in these  file: 
+* [./docs/SETUP-FULL.md](./docs/SETUP-FULL.md) if it is your first time setting up a Spring/React app with Auth0 and Google
+* [./docs/SETUP-QUICKSTART.md](./docs/SETUP-QUICKSTART.md) if you've done these steps before.
+
+## Setting up GitHub Actions (CI/CD, CodeCov)
+
+To setup GitHub Actions so that the tests pass, you will need to configure
+some _secrets_ on the GitHub repo settings page; see: [./docs/github-actions-secrets.md](./docs/github-actions-secrets.md) for details.
+
+This file also describes the setup for Codecov
+
 ## Property file values
 
 This section serves as a quick reference for values found in either [`secrets-localhost.properties`](./secrets-localhost.properties) and/or [`secrets-heroku.properties`](./secrets-heroku.properties).
 
 | Property name                                                     | Heroku only? | Explanation                                                               |
 | ----------------------------------------------------------------- | ------------ | ------------------------------------------------------------------------- |
-| `app.namespace`                                                   |              | See `Getting Started` below                                               |
+| `app.namespace`                                                   |              | See `Getting Started`                                                |
 | `app.admin.emails`                                                |              | A comma separated list of email addresses of permanent admin users.       |
 | `app.member.hosted-domain`                                        |              | The email suffix that identifies members (i.e. `ucsb.edu` vs `gmail.com`) |
 | `app.ucsb.api.consumer_key`                                        |              | The "consumer key" from the site <https://developer.ucsb.edu>; see below for more information. |
 | `spring.data.mongodb.uri` |  | The URL for read only access to the MongoDB database with archived course data; see more information below. |
-| `auth0.domain`                                                    |              | See `Getting Started` below                                               |
-| `auth0.clientId`                                                  |              | See `Getting Started` below                                               |
-| `security.oauth2.resource.id`                                     |              | Should always be `${app.namespace}/api`                                   |
+| `auth0.domain`                                                    |              | See `Getting Started`                                                |
```

Comment:
Might want to change the references to the "Getting Started" section to links to the setup docs or something, since that section has been removed from this document